arch/risc-v/src/common/riscv_schedulesigaction.c

```diff
@@ -37,6 +37,18 @@
 #include "riscv_internal.h"
 #include "riscv_arch.h"
 
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* Format output with register width and hex */
+
+#ifdef CONFIG_ARCH_RV32
+#  define PRIxREG "%08"PRIxPTR
```

Comment:
I recommend to leave `"%"` in format string and move here to

```suggestion
#  define PRIxREG "08" PRIxPTR
```
Same in other places

Reply:
Done